### ISSUE TITLE
Add --trash control to move file to trash, no confirmation

### ISSUE THREAD

**Issue by sebastian-65** [quodlibet/quodlibet#4602]
When doing triage of new music in the library, it would be useful to have some easy way to delete audio file I'm just listetning to (with window hidden) and decided to never listen again, and do it fast without interrupting the flow (playing next song).

- Some parameter such as `--trash` or `--move-to-trash` would help (bindable to custom hotkey)
- File would be moved to Trash
- No confirmation would be needed
- Next song will be played (same behavior is already in UI)

To my knowledge, there's no such way of doing this now.

**Comment by sebastian-65** [comment]
@declension Binded as keyboard shortcut in OS directly, to any hotkey of preference.

**Comment by Meriipu** [comment]
perhaps if a menu or settings (possibly advanced/hidden by default) to toggle on or off different prompts was added you could just use the normal delete/trash functionality without a prompt

since multiple different files can be selected in multiple browsers it could cause some confusing situations about which selection would be removed from the `quodlibet --trash` command

**Comment by sebastian-65** [comment]
@Meriipu Disabling confirmation prompts might help, yet in my scenario there's still no way to assign trash action to hotkey (AFAIK). Having multiple browsers is also good point, I don't have answer to that since I always use 1 browser at once at all times.

(I've edited my post for higher clarity.)

**Comment by Meriipu** [comment]
>there's still no way to assign trash action to hotkey

for what it is worth that has a hotkey of `ctrl+delete` already (I do not think remove from library has a hotkey though).

**Comment by sebastian-65** [comment]
Yes, that's only when you are in the UI, having window opened.  So if you listen to music while you're let's say working, now you have need to do:

- open window first
- hit `ctrl+delete`
- confirm action (popup) -> file gets deleted, next file is playing
- hide window again
- ...and find the place where you left off

That's why it would be so great to replace this with a single hotkey hit.

**Comment by sebastian-65** [comment]
@Meriipu Good point, that might be a solid workaround. Thanks for that!

**Comment by declension** [comment]
Slightly related: #4063 (as in - having undo would be nicer way to do this whole thing anyway)
